doc/Data Plane Code Generation/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -1,201 +1,168 @@
-
-**Learn more:** the [autorest.csharp README](https://github.com/Azure/autorest.csharp#setup) has great samples showing how to add convenience APIs in the generated code. Explore this further as you design APIs for your own service.
-
-Here is the step by step process to generate code:
+#### 2. dotnet new project
+- create project folder 'Azure.<group>.<service>'. e.g. Azure.IoT.DeviceUpdate under sdk/<service> folder
+- new project: navigate to the project folder, and run 'dotnet new' as following:
+  
+```
+sdk\<your-service-name>\Azure.<group>.<service>> dotnet new dataplane --libraryName [Client-Library-Title] --swagger [input-swagger-file-path] --securityScopes [security-scopes] --force
+```
+e.g.
+```
+dotnet new dataplane --libraryName DeviceUpdate --swagger https://github.com/Azure/azure-rest-api-specs/blob/23dc68e5b20a0e49dd3443a4ab177d9f2fcc4c2b/specification/deviceupdate/data-plane/Microsoft.DeviceUpdate/preview/2021-06-01-preview/deviceupdate.json --securityScopes https://api.adu.microsoft.com/.default --force
+```
 
-#### 1. Set the Swagger File
+#### 3. update the solution file if needed
+Run 'dotnet sln' as following to update the projects in the solution file:
 
-Open autorest.md in your `Azure.<group>.<service>` project. In the input-file definition, replace the file path with your swagger link, for example: [WebPubSub swagger link](https://github.com/Azure/azure-rest-api-specs/blob/39c7d63c21b9a29efe3907d9b949d1c77b021907/specification/webpubsub/data-plane/WebPubSub/stable/2021-10-01/webpubsub.json). Update a namespace definition to set the namespace C# generator will use when creating your types. Update the AAD token credential(for AAD token, refer [this](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template-dpg/Azure.Template.Generated/src/autorest.md#autorest-configuration) example) or Azure key credential (for Azure Key credential, refer [this](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/webpubsub/Azure.Messaging.WebPubSub/src/autorest.md#swagger-sources) example). Save your changes.
+```
+dotnet sln remove src\Azure.<group>.<service>.csproj
+dotnet sln add src\Azure.<group>.<service>.csproj
+dotnet sln remove tests\Azure.<group>.<service>.Tests.csproj
+dotnet sln add tests\Azure.<group>.<service>.Tests.csproj
+```
 
-Your `autorest.md` file should now look like:
+**Learn more:** to understand more about the Azure SDK repo structure, see [Repo Structure](https://github.com/Azure/azure-sdk/blob/master/docs/policies/repostructure.md) in the `azure-sdk` repo.
 
-````md
-# `Azure.<group>.<service>` Code Generation
+### Generate client library
```

Comment:
This is a step we want the reader to take, whereas **"Create the project folder"** was something the script does for them.  Would it be possible to update the formatting so this is more clear?

Reply:
The script Invoke-DataPlaneGenerateSDKPackage.ps1 will do everything of the starting package.  This chapter is to describe what the script does.
I add a statement 'The script `eng\scripts\automation\Invoke-DataPlaneGenerateSDKPackage.ps1` will do **step-by-step** as following.' above to indicate.